README.md

```diff
@@ -138,9 +138,9 @@ $faker->ext(PersonDe::class)->firstName(); // German name
 use Faker\Factory;
 
 $faker = Factory::create();
-$faker->withUnique()->name();
-$faker->withMaybe(0.8)->name();
-$faker->withValid(fn($v) => strlen($v) > 3))->name();
```

Comment:
The `with` prefixes make sense because people who are aware of the pattern will understand that invoking the methods does not modify the generator, but will return a new generator, but perhaps it's a bit too much?

Reply:
Yeah, I will return a new Generator. So if you do:

```php
$faker = $faker->withMaybe(0.5);
$faker->name(); // There is a 50% chance you dont get a name
$faker->foo(); // Same here 50%
```

Having "with" makes that more explicit.